pkg/controllers/cloud/vpc_cache.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2022 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloud
+
+import (
+	"fmt"
+
+	"github.com/go-logr/logr"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/cache"
+
+	runtimev1alpha1 "antrea.io/nephe/apis/runtime/v1alpha1"
+)
+
+type VpcCacheData struct {
+	VpcIndexer cache.Indexer
```

Comment:
Why have VpcCacheData?? Do you forsee to store other stuff in it. Cache indexer is all we care about

Reply:
My bad, i had something else too when started with it, thats why created a struct, now that we do not need anything else other cache indexer, we don't need to have a struct, remove dit latest patch.